app/Server.hs

```diff
@@ -206,7 +206,8 @@ configParser =
    some
      (strOption
         (long "package" <> short 'p' <> metavar "PATH" <>
-         help "Path to a Cabal package"))) <*>
+         help "Path to a Cabal package (Default: '.')"))
+   <|> pure (Directories ["."])) <*>
```

Comment:
Not using `value` for this one because then `some` will always succeed and we'll get an infinite loop.